### PR TITLE
Add webpack configuration for React setup

### PR DESCRIPTION
**Pull request by gunjan2012**
Hello @manas-vyas-30 ,
This is react setup which is created using webpack from scratch.With this setup, you can easily bundle your React application and all its dependencies into a single file, and run it in the browser.

The webpack configuration includes the following features:
1. Support for JSX syntax and ES6 modules
2. Development server with hot reloading
3. Production build with minification and optimization

To use this setup, clone this repository and run the following commands:
- npx @gunjan_2001/create-react-app-using-webpack YOUR-PROJECT-NAME
- npm run start # for development
- npm run build # for production

Please review this pull request and let me know if you have any feedback or suggestions. Thank you!

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+# Dependency directories
+node_modules
```
Comment (manas-vyas-30) [inline .gitignore:2]: Should have added `build` in .gitignore file.

**Review by manas-vyas-30 — APPROVED** [review]
@gunjan2012  Good job, React Boilerplate looks good. Sharing these comments for your reference. Please go through them.

**Review by manas-vyas-30 — COMMENTED** [review]
`node_modules` should not come with project setup when user setups project using `npx @gunjan_2001/create-react-app-using-webpack YOUR-PROJECT-NAME` command.

**Comment by gunjan2012** [comment]
> `node_modules` should not come with project setup when user setups project using `npx @gunjan_2001/create-react-app-using-webpack YOUR-PROJECT-NAME` command.

Hi @manas-vyas-30 , It is node version issue this setup will work in node version 14 and above, i have add one error message that this setup will work in node version 14 and above so now it will give this error message when user will run this command .

**Pull request merged**